src/opera_utils/_helpers.py

```diff
@@ -37,3 +37,14 @@ def reproject_bounds(bounds: Bbox, src_epsg: int, dst_epsg: int) -> Bbox:
     left, bottom, right, top = bounds
     bbox: Bbox = (*t.transform(left, bottom), *t.transform(right, top))  # type: ignore
     return bbox
+
+
+def reproject_coordinates(
+    x: list[float], y: list[float], src_epsg: int, dst_epsg: int
```

Comment:
How about...
```suggestion
    x: Sequence[float], y: Sequence[float], src_epsg: int, dst_epsg: int
```
that way we can use the function with a tuple or a numpy array of x/y coordinates, and mypy won't yell at us

(we'll need to add `from typing import Sequence` up top as well